src/main/java/io/vertx/redis/client/RedisConnection.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ * <p>
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ * <p>
+ * The Eclipse Public License is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * <p>
+ * The Apache License v2.0 is available at
+ * http://www.opensource.org/licenses/apache2.0.php
+ * <p>
+ * You may elect to redistribute this code under either of these licenses.
+ */
+package io.vertx.redis.client;
+
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.codegen.annotations.Nullable;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.*;
+import io.vertx.core.streams.ReadStream;
+
+import java.util.List;
+
+/**
+ * A simple Redis client.
+ */
+@VertxGen
+public interface RedisConnection extends ReadStream<Response> {
```

Comment:
I think this class should not copy the javadoc from ReadStream and instead inherit the doc from the parent interface.

also it should use `@Override` annotation on methods